src/api/domain/community.api.server.ts

```diff
@@ -0,0 +1,5 @@
+import { privateApi } from '../config/privateApi.server';
+
+export const getCommunityIdCard = async (id: number) => {
+  return privateApi.get(`communities/${id}/idCards`);
+};
```

Comment:
path가 잘못 들어간 것으로 보입니다!

```suggestion
export const getCommunityIdCard = async (id: number) => {
  return privateApi.get(`/communities/${id}/idCards`);
};
```

Reply:
그러네요! 확인 감사합니다!